vhdbuilder/packer/install-dependencies.sh

```diff
@@ -402,27 +402,27 @@ done
 # need to cover previously supported version for VMAS scale up scenario
 # So keeping as many versions as we can - those unsupported version can be removed when we don't have enough space
 # below are the required to support versions
-# v1.15.11-hotfix.20200714.1
-# v1.15.12-hotfix.20200714.1
-# v1.16.10-hotfix.20200714.1
-# v1.16.13-hotfix.20200714.1
-# v1.17.7-hotfix.20200714.2
-# v1.17.9-hotfix.20200714.1
+# v1.15.11-hotfix.20200817.1
```

Comment:
Windows does not patch 1.15.11 and 1.15.12 because they will be retired soon.

Reply:
I know. I'm doing this as it's a free lunch to me.

Reply:
We should follow the same policy for both Windows and Linux. If we pick these changes for Linux, we should also pick these for Windows.